girder/api/rest.py

```diff
@@ -558,7 +558,7 @@ def _handleAccessException(e):
         cherrypy.response.status = 401
     else:
         cherrypy.response.status = 403
-        logger.exception('403 Error')
+        logger.debug('403 Error')
```

Comment:
This represents a policy change that should be considered carefully by Girder stakeholders. I don't think just changing this to `debug` is the correct solution. A better solution might be to divert 403 errors into a separate logging channel.

I think we should separate out the `propagate = False` change from this one, since I think we want to merge that one quickly and it's probably not controversial.

Reply:
https://docs.python.org/3/library/logging.html#logging.Logger.exception
> Logs a message with level ERROR on this logger. The arguments are interpreted as for debug(). Exception info is added to the logging message. This method should only be called from an exception handler.

Girder denying access to a resource is not an application-level ERROR, and should not be logged with that severity.

If you'd like a separate logging channel for 403 or similar security-adjacent events, that sounds like a great new feature, but is out of scope of this PR and the main `girder` logger.

Reply:
We almost don't use log level `debug` in Girder (it looks like it only gets used in a few places in the filesystem assetstore).  Perhaps we want to log this at the `info` level.

Reply:
> If you'd like a separate logging channel for 403 or similar security-adjacent events, that sounds like a great new feature, but is out of scope of this PR and the main girder logger.

Since 403 events aren't "supposed" to happen under normal operation of Girder, we log them because they may represent malicious attacks, and the inclusion of their stack trace is important in that regard. I agree that they don't belong in the same channel as 500-level exceptions since they're client errors and not server errors. However, I don't want to merge this part until we develop the alternative place to put them, stack trace included; I don't want to create a gap where these things went from being logged *somewhere* to not being logged at all (setting them as "debug" events is essentially throwing them away). I contend that `exception` is the appropriate level for the event, since an exception was thrown and we are handling it, and the exception information is desired in the log.